packages/flutter/test/material/chip_test.dart

```diff
@@ -2792,6 +2792,51 @@ void main() {
     expect(labelStyle.style.color, equals(Colors.black.withAlpha(0xde)));
   });
 
+  testWidgets('Chip uses the resolved color based on its state for the delete icon color', (
+    WidgetTester tester,
+  ) async {
+    final ThemeData themeData = ThemeData(primarySwatch: Colors.blue);
```

Comment:
nit: this seems unnecessary. I think we can remove it to make test look a bit simpler.
```suggestion
```